.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Run Checks
+on:
+  push:
+    branches:
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint code base
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Config NodeJS dependency caching
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: "12"
```

Comment:
Any reason we went with node js version 12?

Reply:
This is the version we use in our env for this code